app/assets/javascripts/discourse/app/services/desktop-notifications.js

```diff
@@ -42,8 +43,12 @@ export default class DesktopNotificationsService extends Service {
       : false;
   }
 
+  get isSupported() {
```

Comment:
I'd like to offer another getter function that returns the result of `isPushNotificationsSupported`, but I'm not sure how to write an `async` getter function (and the codebase has no existing examples).